frontend/src/components/Navbar.jsx

```diff
@@ -10,6 +9,9 @@ export default function Navbar() {
       <AppBar position="static">
         <Toolbar>
           <Button color="inherit">Login</Button>
+          <Button color="inherit">Login</Button>
```

Comment:
maybe make the buttons say different things, plan out what buttons we will need perhaps